src/bots/uncrossArbBot.ts

```diff
@@ -212,45 +437,47 @@ export class UncrossArbBot implements Bot {
 						PRICE_PRECISION
 					);
 
-					const bidMakerInfo: MakerInfo = {
-						makerUserAccount: this.orderSubscriber.usersAccounts.get(
-							bestDriftBid.userAccount!.toBase58()
-						)!.userAccount,
-						order: bestDriftBid.order,
-						maker: bestDriftBid.userAccount!,
-						makerStats: getUserStatsAccountPublicKey(
-							this.driftClient.program.programId,
-							this.orderSubscriber.usersAccounts.get(
-								bestDriftBid.userAccount!.toBase58()
-							)!.userAccount.authority
-						),
-					};
-
-					const askMakerInfo: MakerInfo = {
-						makerUserAccount: this.orderSubscriber.usersAccounts.get(
-							bestDriftAsk.userAccount!.toBase58()
-						)!.userAccount,
-						order: bestDriftAsk.order,
-						maker: bestDriftAsk.userAccount!,
-						makerStats: getUserStatsAccountPublicKey(
-							this.driftClient.program.programId,
-							this.orderSubscriber.usersAccounts.get(
-								bestDriftAsk.userAccount!.toBase58()
-							)!.userAccount.authority
-						),
-					};
-
-					console.log('best ask', bestDriftAsk.userAccount!.toBase58());
-					console.log('best bid', bestDriftBid.userAccount!.toBase58());
-
 					const midPrice = (bestBidPrice + bestAskPrice) / 2;
 					if (
 						(bestBidPrice - bestAskPrice) / midPrice >
 						2 * driftUser.getMarketFees(MarketType.PERP, perpIdx).takerFee
 					) {
+						let bidMakerInfo: MakerInfo;
```

Comment:
think we should log the market, the users, there size and price in here so it's easier to debug

Reply:
done